packages/base-styles/_mixins.scss

```diff
@@ -479,186 +479,48 @@
 	}
 }
 
-@mixin background-colors() {
-	.has-pale-pink-background-color {
-		background-color: #f78da7;
-	}
-
-	.has-vivid-red-background-color {
-		background-color: #cf2e2e;
-	}
-
-	.has-luminous-vivid-orange-background-color {
-		background-color: #ff6900;
-	}
-
-	.has-luminous-vivid-amber-background-color {
-		background-color: #fcb900;
-	}
-
-	.has-light-green-cyan-background-color {
-		background-color: #7bdcb5;
-	}
-
-	.has-vivid-green-cyan-background-color {
-		background-color: #00d084;
-	}
-
-	.has-pale-cyan-blue-background-color {
-		background-color: #8ed1fc;
-	}
-
-	.has-vivid-cyan-blue-background-color {
-		background-color: #0693e3;
-	}
-
-	.has-vivid-purple-background-color {
-		background-color: #9b51e0;
-	}
-
-	.has-white-background-color {
-		background-color: #fff;
-	}
-
-	// Deprecated from UI, kept for back-compat.
+// Deprecated from UI, kept for back-compat.
+@mixin background-colors-deprecated() {
 	.has-very-light-gray-background-color {
 		background-color: #eee;
 	}
 
-	.has-cyan-bluish-gray-background-color {
-		background-color: #abb8c3;
-	}
-
-	// Deprecated from UI, kept for back-compat.
 	.has-very-dark-gray-background-color {
 		background-color: #313131;
 	}
-
-	.has-black-background-color {
-		background-color: #000;
-	}
 }
 
-@mixin foreground-colors() {
-	.has-pale-pink-color {
-		color: #f78da7;
-	}
-
-	.has-vivid-red-color {
-		color: #cf2e2e;
-	}
-
-	.has-luminous-vivid-orange-color {
-		color: #ff6900;
-	}
-
-	.has-luminous-vivid-amber-color {
-		color: #fcb900;
-	}
-
-	.has-light-green-cyan-color {
-		color: #7bdcb5;
-	}
-
-	.has-vivid-green-cyan-color {
-		color: #00d084;
-	}
-
-	.has-pale-cyan-blue-color {
-		color: #8ed1fc;
-	}
-
-	.has-vivid-cyan-blue-color {
-		color: #0693e3;
-	}
-
-	.has-vivid-purple-color {
-		color: #9b51e0;
-	}
-
-	.has-white-color {
-		color: #fff;
-	}
-
-	// Deprecated from UI, kept for back-compat.
+// Deprecated from UI, kept for back-compat.
+@mixin foreground-colors-deprecated() {
 	.has-very-light-gray-color {
 		color: #eee;
 	}
 
-	.has-cyan-bluish-gray-color {
-		color: #abb8c3;
-	}
-
-	// Deprecated from UI, kept for back-compat.
 	.has-very-dark-gray-color {
 		color: #313131;
 	}
-
-	.has-black-color {
-		color: #000;
-	}
 }
 
-@mixin gradient-colors() {
-	// Our classes uses the same values we set for gradient value attributes, and we can not use spacing because of WP multi site kses rule.
+// Deprecated from UI, kept for back-compat.
+@mixin gradient-colors-deprecated() {
```

Comment:
Upon reviewing the list with fresh eyes today, I've found that there were some gradient colors that are deprecated (no longer in the list of gradients we offer but need to keep in case they had been used). Added them.